SECURITY.md

```diff
@@ -0,0 +1,37 @@
+# Security Policy
+
+## Supported Versions
+
+Security-related reports are considered for official numbered releases
+starting with v3.5. However, issues that do not affect the current Stable or
+Beta series are unlikely to be fixed. Please see
+http://www.squid-cache.org/Versions/ for the list of releases that belong to
+the current series.
+
+Reports about security issues in the Development series are welcomed. However,
+development series contains experimental code that does not qualify for CVE
+allocation under MITRE policy.
+
+
+## Reporting a Vulnerability
+
+To report security-sensitive bugs, please post to the squid-bugs mailing
+(list)[http://www.squid-cache.org/Support/mailing-lists.html#squid-bugs]. It
+is a closed list (although anyone can post), and security related bug reports
+are treated in confidence until the impact has been established.
+
+The security team strives to manually acknowledge each new report within 48
+hours. Please feel free to email a reminder if you have not heard from us
+within that time frame.
+
+As a _last_ resort (e.g., if the squid-bugs contact point appears to be
+broken), contact the release maintainer directly. The maintainer is on the
+security team but may not be able to respond promptly.
+
+
+### Encrypted reports
+
+Reporters wishing to encrypt their vulnerability reports can request GPG
+public keys from the security team members via the squid-bugs mailing list.
```

Comment:
This procedure is rather unusual and probably relatively insecure. Perhaps we can quickly create and publish a "security team" GPG key on the Squid web site instead? I know other open source projects use such team keys for this very purpose. I do not insist on this change, of course.

Reply:
This is a dump of what we have been doing so far. It has issues, but security ones are not necessarily present.

FYI; when requested I provide a private key attached to my maintainer email contact which has been mutually counter-signed by the official Squid Release key and can be validated in all the normal ways for a GPG key. I assume the one you provide has suitable web-of-trust as well.

Lets discuss a team key on-list instead of here.